apps/site/src/components/dom/Layout.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { ReactNode, useRef } from "react";
+import dynamic from "next/dynamic";
+const Scene = dynamic(() => import("@/components/canvas/Scene"), {
+	ssr: false,
+});
+
+const Layout = ({ children }: { children: ReactNode }) => {
+	const ref = useRef<any>();
```

Comment:
Nitpick: Since the ref is for a `div` you should say `useRef<HTMLDivElement>(null)` instead.